docs/user_guides/fs/tags/tags.md

```diff
@@ -0,0 +1,138 @@
+# Tags 
+
+## Introduction
+
+Hopsworks feature store enables users to attach tags to artifacts, such as feature groups, feature views or training datasets.
+
+A tag is a `{key: value}` association which provides additional information about the data managed by Hopsworks. Tags allow organizations to customize the feature store metadata and build a governance layer on top of it. Hopsworks indexes tags and makes them searchable by users on the platform, increasing the discoverability of features and artifacts in the feature store.
+
+## Prerequisites
+
+Tags need need to respect a given schema. Before being able to attach a tag to an artifact you need to define a schema.
```

Comment:
"need need" 
Also, use the active voice.
I would phrase this differently:
Tags have a schema. Before you can attach a tag to an artifact and fill in the tag values, you first need to select an existing tag schema or create a new tag schema.